components/NavigationDrawer/src/MDCBottomDrawerPresentationController.m

```diff
@@ -246,6 +246,22 @@ - (void)setTopHandleColor:(UIColor *)topHandleColor {
   self.topHandle.backgroundColor = topHandleColor;
 }
 
+- (MDCShadowElevation)elevation {
+  return self.bottomDrawerContainerViewController.elevation;
```

Comment:
I could be misreading the code, but it looks like `self.bottomDrawerContainerViewController` is `nil` until `presentationTransitionWillBegin` is called.  In that case, does setting the property before the transition begins result in no value actually being set?

Reply:
that's true, updated accordingly